src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4098,7 +4098,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <ItemGroup>
       <_SatelliteAssemblies Include="@(IntermediateSatelliteAssembliesWithTargetPath);@(ReferenceSatellitePaths)" />
       <_DeploymentReferencePaths Include="@(ReferenceCopyLocalPaths)"
-                                 Condition="'%(Extension)' == '.dll' Or '%(Extension)' == '.exe'">
+                                 Condition="'%(Extension)' == '.dll' Or '%(Extension)' == '.exe' Or '%(Extension)' == '.md'">
```

Comment:
what is that change? md file?

Reply:
.NET Core Apps runtime pack also includes a .md file. This .md file need to be published in self-contained deployment mode.

The change in the line is to ensure the .md file which gets published in the self contained deployment mode is also included in ClickOnce's manifest file.

Reply:
did you test with a project has md files?

Reply:
self contained/single files/read to run?

Reply:
We'll be sure to test the scenario. That said, the line changed here is for ReferencePaths and I don't think .md files can be added explicitly as references. If they can and have copylocal set to true, then we will need to publish it.

Reply:
I have verified that .md cannot be added as a reference. If the project has a .md file as a data file with copylocal set to true, we will publish the .md as a data file fine. This however is unrelated to my change which is only specific to References and the only way a .md reference can come in is through the runtime pack.